src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/energy/EnergyNet.java

```diff
@@ -261,10 +271,10 @@ private int tickAllCapacitors() {
     private void updateHologram(@Nonnull Block b, double supply, double demand) {
         if (demand > supply) {
             String netLoss = NumberUtils.getCompactDouble(demand - supply);
-            updateHologram(b, "&4&l- &c" + netLoss + " &7J &e\u26A1");
+            updateHologram(b, "&4&l- &c" + netLoss + " &7J &e⚡");
```

Comment:
Not really sure I like this change? Can’t say I even have a reason to voice, however.

Reply:
I did this for consistency as I saw it like this in some other places, but upon further looking it looks like that was caused by my decompiler just replacing it automatically. Will revert 👍